src/html/pages/_header.ejs

```diff
@@ -21,6 +21,26 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
 <link href="<%= assetUrl('bundle.css') %>" rel="stylesheet">
 <title>Hylo</title>
+
+<script type="text/javascript">
+
+    (function(b,r,a,n,c,h,_,s,d,k){if(!b[n]||!b[n]._q){for(;s<_.length;)c(h,_[s++]);d=r.createElement(a);d.async=1;d.src="https://cdn.branch.io/branch-v1.7.0.min.js";k=r.getElementsByTagName(a)[0];k.parentNode.insertBefore(d,k);b[n]=h}})(window,document,"script","branch",function(b,r){b[r]=function(){b._q.push([r,arguments])}},{_q:[],_v:1},"addListener applyCode banner closeBanner creditHistory credits data first getCode init link logout redeem referrals removeListener sendSMS setIdentity track validateCode".split(" "), 0);
+
+    branch.init('key_live_mhdVu4UFbZnZ86jxdBGyjbajqubGzCLQ')
+    branch.banner(
+      {
+        icon: '/img/appicon.png',
+        title: 'Hylo App',
+        description: 'The Hylo App!',
+        forgetHide: 10,
+        mobileSticky: true,
+        showDesktop: false
+      },
+      {}
+    )
```

Comment:
@levity love to hear if you have any ideas about reducing the duplication between this and src/js/app/index.js